tests/parsers/beckman_vi_cell_xr/testdata/v2.06/style_fill_incorrect.json

```diff
@@ -71,7 +71,7 @@
             "equipment serial number": "AL42244"
         },
         "data system document": {
-            "file name": "style_fill_error.xlsx",
+            "file name": "style_fill_incorrect.xlsx",
```

Comment:
NOTE to reviewer: I had to rename this file because the "error" in the name was causing the auto-test discovery to skip it.